frontend/src/app/shared/export-item-modal/export-item-modal.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="modal-header">
+  <h4 class="modal-title">Export option</h4>
+</div>
+<div class="modal-body">
+  <div>
+    <label class="col form-label" for="delimiter">Delimiter</label>
+    <select name="delimiter" class="col form-select w-50">
+      @for (option of delimiterOptions; track option) {
+        <option [value]="option">{{ option }}</option>
+      }
```

Comment:
You should use an enum translation (see other example in the code) to display "Semi-Colon ;" instead of just ";"